src/components/02-components/card-pending-tasks/card-pending-tasks.tsx

```diff
@@ -0,0 +1,63 @@
+import { Button, Paper, Typography } from '@mui/material';
+import { useTheme } from '@mui/material/styles';
```

Comment:
Please change this to

```
import { useTheme } from '@mui/material';
```

There's a bug when the theme is important in the other way.